docs/index.rst

```diff
@@ -14,14 +14,14 @@ or interactive CLI tools, please refer to the `Rossum package <https://github.co
 Key Features
 ------------
 
-* **Pagination handling**: Automatically merges paginated results into one list
+* **Pagination handling**: Automatically merges paginated results into a single list
```

Comment:
I would order the list by importance. Pagination is not exactly the killer feature I think. Same goes for the same list in README.md